src/controllers/hotelController.js

```diff
@@ -0,0 +1,41 @@
+import hotelModel from "../models/hotelModel";
+
+class Hotel {
+  constructor() {
+    //
+  }
+
+  async createHotel(req, res) {
```

Comment:
from the last review, you were meant to remove the word "Hotel" from all methods.

remove the word "Hotel" from all methods so your methods names will simply be like so... create, get, getById, and so on..